pkg/controllers/awsloadbalancercontroller/controller.go

```diff
@@ -38,7 +40,11 @@ import (
 	"github.com/openshift/aws-load-balancer-operator/pkg/aws"
 )
 
-const allowedResourceName = "cluster"
+const (
+	controllerName               = "cluster"
+	controllerSecretName         = "aws-load-balancer-operator-credentials"
+	controllerServiceAccountName = "aws-load-balancer-operator-controller-manager"
```

Comment:
These seem to be the operand secret and service account, it's slightly misleading to see `operator` in their names.

Reply:
Fixed in accordance with [this](https://github.com/openshift/aws-load-balancer-operator/pull/10#discussion_r819339071)